File: Sort 0s, 1s and 2s.java
class Solution {
    // Function to sort an array of 0s, 1s, and 2s
    public void sort012(ArrayList<Integer> arr) {
        int a[] = new int[arr.size()];
        
        // Copy elements from ArrayList to array
        for (int i = 0; i < arr.size(); i++) {
            a[i] = arr.get(i);
        }
        
        // Sort the array
        Arrays.sort(a);
        
        // Copy sorted elements back to the ArrayList
        for (int i = 0; i < arr.size(); i++) {
            arr.set(i, a[i]);
        }
    }
}

File: Sort 0s, 1s and 2s.cpp
class Solution {
  public:
    void sort012(vector<int>& arr) {
        // code here
         int n = arr.size();
         int i=0;
         int j=n-1;

         for(int k=0;k<=j;k++){
               if (arr[k]==0){
                    swap(arr[i],arr[k]);
                    i++;
               }
               else if (arr[k]==2){
                    swap(arr[j],arr[k]);
                    j--;
                          k--;  
               }
         }
    

    }
};
